src/Compiler/Driver/CompilerOptions.fs

```diff
@@ -2410,6 +2438,11 @@ let ReportTime (tcConfig: TcConfig) descr =
 
         tPrev <- Some(tStart, tNow, utNow, gcNow)
 
+    nPrev
```

Comment:
Sooo why this change?

Reply:
The activity must be .Dispose()'d only after the tags are enhanced (see above), not before.
But anyway, I have code lined up that removes this full function and reuses the generic Activity mechanism (a dedicated listeners for console logging).